Week3/app.js

```diff
@@ -0,0 +1,239 @@
+  }
+};
+print();
+
+// 4.Create a function named vehicleType that receives a color, and a code, 1 for car, 2 for motorbike. And prints 'a blue motorbike' for example when called as vehicleType("blue", 2)
+
+const vehicleType = (color, code) => {
+  if (code === 1) {
+    console.log(`a ${color} car`);
+  } else if (code === 2) {
+    console.log(`a ${color} motorbike`);
+  } else {
+    console.log('nothing');
+  }
+};
+vehicleType('yellow', 2);
+
+/* 5. Can you write the following without the if statement, but with just as a single line with console.log(...);?
+ if (3 === 3) {
+    console.log("yes");
+  } else {
+    console.log("no");
+  }
+ */
+console.log(3 === 3 ? `yes` : `no`);
+
+// 6.Create a function called vehicle, like before, but takes another parameter called age, so that vehicle("blue", 1, 5) prints 'a blue used car'
+
+const vehicle = (color, code, age) => {
+  const newa = age === 0 ? `new` : `used`;
+
+  if (code === 1) {
+    console.log(`a ${color} ${newa} car`);
+  } else if (code === 2) {
+    console.log(`a ${color} ${newa} motorbike`);
+  }
+};
+vehicle('blue', 1, 1);
+
+// 7. Make a list of vehicles, you can add "motorbike", "caravan", "bike", or more.
+const vehicles = ['motorbike', 'caravan', 'bike', 'car'];
+
+//8. How do you get the third element from that list?
+
+console.log(vehicles[2]);
+
+//9. Change the function vehicle to use the list of question 7. So that vehicle("green", 3, 1) prints "a green new bike".
+
+let vehic = (color, code, age) => {
+  age === 1 || age === 0 ? (age = 'new') : (age = 'used');
+  const type = vehicles[code];
+  console.log(`a ${color} ${age} ${type}`);
+};
+
+vehic('green', 2, 1);
+
+// 10. Use the list of vehicles to write an advertisement. So that it prints something like: "Amazing Joe's Garage, we service cars, motorbikes, caravans and bikes."
+
+const vehicless = ['motorbike', 'car', 'caravan', 'bike'];
+
+newprint = () => {
+  const first = vehicless[3];
+
+  const second = vehicless
+    .map(vehicless => vehicless + 's')
+    .filter((add, index) => add !== first + 's')
+    .join(', ');
+
+  console.log(`Amazing Joe's Garage, we service ${second}  and ${first}s.`);
+};
+
+newprint();
+
+// 11. What if you add one more vehicle to the list, can you have that added to the advertisement without changing the code for question 10?
+
+vehicless.push('boat');
+console.log(vehicless);
+
+// 12.Create an empty object.
+const empty = {};
+
+// 13.Create an object that contains the teachers that you have had so far for the different modules.
+
+const teacherlesson = [
+  {
+    name: 'Philipp Beau',
+    courses: 'HTML, CSS',
+  },
+  {
+    name: 'Rob van Kruijsdijk',
+    courses: 'HTML, CSS',
+  },
+
+  {
+    name: 'Bonan Zhao',
+    courses: 'CLI, Git',
+  },
+  {
+    name: 'Yash Kapila',
+    courses: 'Git, GitHub, JavaScript',
+  },
+];
+console.log(teacherlesson);
+
+// 17.What does the following code return? (And why?)
```

Comment:
Did you forget assignment `15` and `16`?

Reply:
Actually, I didn't forget assignments 15 and 16. I had sent both of previously homework that's why  I didn't send again but if you want I could be sent

Reply:
Ah okay:) then it's fine!